services/wfm/src/test/java/org/openkilda/wfm/isl/DiscoveryManagerTest.java

```diff
@@ -337,31 +342,28 @@ public void handleSwitchUp() {
         dm.handleDiscovered(srcNode3.getDatapath(), srcNode3.getPortNumber(),
                 dstNode3.getDatapath(), dstNode3.getPortNumber());
 
-        List<DiscoveryLink> links;
+        Set<DiscoveryLink> links;
```

Comment:
We could define links in one line:
`Set<DiscoveryLink> links = dm.findAllBySwitch(srcNode1.getDatapath());`

Reply:
Fixed.